diff --git a/node_modules/@react-native-community/async-storage/android/build.gradle b/node_modules/@react-native-community/async-storage/android/build.gradle
index 4dc43dc..131f6be 100644
--- a/node_modules/@react-native-community/async-storage/android/build.gradle
+++ b/node_modules/@react-native-community/async-storage/android/build.gradle
@@ -38,6 +38,7 @@ apply plugin: 'com.android.library'
 android {
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
+    namespace 'com.reactnativecommunity.asyncstorage'
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 19)
         targetSdkVersion safeExtGet('targetSdkVersion', 28)
diff --git a/node_modules/@react-native-community/async-storage/android/src/main/AndroidManifest.xml b/node_modules/@react-native-community/async-storage/android/src/main/AndroidManifest.xml
index 7e180bd..5f5d7a2 100644
--- a/node_modules/@react-native-community/async-storage/android/src/main/AndroidManifest.xml
+++ b/node_modules/@react-native-community/async-storage/android/src/main/AndroidManifest.xml
@@ -1,6 +1,4 @@

-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.reactnativecommunity.asyncstorage">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">

 </manifest>
-
\ No newline at end of file
